custom_components/ipx800v4/binary_sensor.py

```diff
@@ -36,26 +40,50 @@ async def async_setup_entry(
 
     for device in devices:
         if device.get(CONF_TYPE) == TYPE_VIRTUALOUT:
-            entities.append(VirtualOutBinarySensor(device, controller, coordinator))
+            on_value = 0 if device.get(CONF_INVERT) else 1
+            entities.append(VirtualOutBinarySensor(device, controller, coordinator, on_value))
```

Comment:
you don't have to handle it here

Reply:
I initially implemented that below but my python fu is so bad that it wasn't working, I'll try to move that down again